.github/workflows/dispatch-labels-hooks-topics-settings.yaml

```diff
@@ -0,0 +1,74 @@
+name: Dispatch labels, hooks, topics and settings
+
+on:
+    schedule:
```

Comment:
Why not run them also on a merge in master?

Reply:
My idea is to add a button on actions to manually trigger them. Normally those changes can wait a few hours, it doesnt matter. The problem with doing it on each merge is that we could easily hit rate limits